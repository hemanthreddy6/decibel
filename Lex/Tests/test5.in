132
1.
.1
1.1
12.
.12
12.12
1.12
12.1
"word1"
'word2'
word3
word4
main
