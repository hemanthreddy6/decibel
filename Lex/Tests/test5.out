INT_LITERAL
FLOAT_LITERAL
FLOAT_LITERAL
FLOAT_LITERAL
FLOAT_LITERAL
FLOAT_LITERAL
FLOAT_LITERAL
FLOAT_LITERAL
FLOAT_LITERAL
STRING_LITERAL
STRING_LITERAL
IDENTIFIER
IDENTIFIER
MAIN
